curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-the-date-object-by-building-a-date-formatter/646b601805ebd90649999f96.md

```diff
@@ -0,0 +1,163 @@
+---
+id: 646b601805ebd90649999f96
+title: Step 24
+challengeType: 0
+dashedName: step-24
+---
+
+# --description--
+
+In the same selector named `select`, add these styles:
```

Comment:
Let's remove this step since this is the fourth project in the curriculum. 

By this point, we are supplying the campers with all of the HTML and CSS.

Reply:
removed